basyx.aasrepository/basyx.aasrepository-core/src/test/java/org/eclipse/digitaltwin/basyx/aasrepository/AasRepositorySuite.java

```diff
@@ -244,4 +245,9 @@ public void getPaginatedAssetAdministrationShellIterating() {
 	public AasService getAasService(AssetAdministrationShell shell) {
 		return new AasRepositoryAasServiceWrapper(getAasRepository(), shell);
 	}
+	
+	@Override
+	public FileRepository getFileRepository() {
```

Comment:
I did this over here to avoid implementing it in to the child classes because for some, there is no FileRepository implementation. Returning null here just allows me to set a default FileRepository in case nothing is present.

Reply:
Fixed.